.github/actions/secure-download-artifact/action.yml

```diff
@@ -2,11 +2,11 @@ name: "Secure artifact download"
 description: "Download an artifact and verify its SHA256"
 inputs:
   name:
-    description: "Artifact name."
+    description: "Artifact name. (Note: this is a name given to an upload, not the path or filename)"
     required: true
   path:
-    description: "Destination path."
-    required: false
+    description: "The path of the artifact, as uploaded by the secure-upload-artifact or actions/upload-artifact GHA."
```

Comment:
I'm not sure I get what "as uploaded by ..." means. Wouldn't this just be the path you download the artifact to in the workspace?

Reply:
You're right it's confusing. This is actually the path of the file for the artifact name. Say, if you used:
```
uses: secure-upload-artifact
with:
  name: my-name
  path: /path/to/upload.txt
```

then you'd need:
```
uses: secure-download-artifact
with:
  name: my-name
  path: /path/to/upload.txt
```

I'll update the description to `as provided to the Actions during upload`